### PR TITLE
Adding a case study implementation of the Model factory

### PR DESCRIPTION
**Pull request by prasann**
This is a sample implementation of model factory. This example is a sanitized version of an implementation we did as part of a client engagement.

Related to this: #17

### CODE REVIEW

File: use_case_implementation/fridge_object_detection/.gitignore [hunk 1]
```diff
@@ -0,0 +1,372 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (ritesh-modi) [inline use_case_implementation/fridge_object_detection/.gitignore:1]: Can we add text in documentation at top level how we used the repo, what changes were done for the implementation

File: use_case_implementation/fridge_object_detection/.gitignore [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x64/
+x86/
+[Ww][Ii][Nn]32/
+[Aa][Rr][Mm]/
+[Aa][Rr][Mm]64/
+bld/
```
Comment (ritesh-modi) [inline use_case_implementation/fridge_object_detection/.gitignore:29]: Can we rename top-level folder as telco-case-study-implementation

**Review by ritesh-modi — CHANGES_REQUESTED** [review]
Couple of comments, rest LGTM

File: use_case_implementation/fridge_object_detection/.gitignore (hunk 1 above)
Comment (prasann) [reply use_case_implementation/fridge_object_detection/.gitignore:1]: I have added this in the Readme inside the implementation folder. `Adaptation from the accelerator and differences` Please let me know whether that will do.

File: use_case_implementation/fridge_object_detection/.gitignore (hunk 2 above)
Comment (prasann) [reply use_case_implementation/fridge_object_detection/.gitignore]: done

**Pull request merged**